src/components/TextField/TextField.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import Grid from '@material-ui/core/Grid';
+import TextField from '@material-ui/core/TextField';
+
+export default function TextFieldForm() {
+    const inputProps = {
+      step: 300,
+    };
+
+    return (<Grid container justify="space-around" alignItems="baseline">
+    <TextField id="standard-basic" label="First" type="text" required="true" autoFocus="true" inputProps={inputProps} />
```

Comment:
Do not return grid instaed of textfield
each property can be set from stories.js

Reply:
Fixed. Please review